nixos/doc/manual/configuration/gpu-accel.xml

```diff
@@ -101,4 +101,93 @@ ROCR_EXT_DIR=`nix-build '&lt;nixpkgs&gt;' --no-out-link -A rocm-runtime-ext`/lib
       </para>
     </section>
   </section>
+
+  <section xml:id="sec-gpu-accel-vulkan">
+    <title>Vulkan</title>
+
+    <para>
+      <link xlink:href="https://en.wikipedia.org/wiki/Vulkan_(API)">Vulkan</link> is a
+      graphics and compute API for GPUs. It is used directly by games or indirectly though
+      compatibility layers like DXVK.
```

Comment:
I think DXVK also warrants a link. There doesn't seem to be a Wikipedia page yet, so maybe:

https://github.com/doitsujin/dxvk/wiki

?